packages/tabs/src/use-tabs.ts

```diff
@@ -57,6 +57,12 @@ export interface UseTabsProps {
    * The id of the tab
    */
   id?: string
+  /**
+   * Performance 🚀:
+   * If `true`, the MenuItem rendering will be deferred
+   * until the menu is open.
```

Comment:
Should probably refer to these as tabs :)

Reply:
Oops... Done!